### ISSUE TITLE
Postman offers no way to export a SINGLE request to a file or whatever

### ISSUE THREAD

**Issue by dinob68** [a85/POSTMan-Chrome-Extension#11493]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

A coworker or someone asks me for a copy of my request to share with them.  Currently, I have no way to quickly copy my request with all the details such as headers, request body, authorization, etc and share with them.
Currently, I can send them curl only but that is not what I want.
It would be nice if I could just right click on a request and copy or export it to a text file.  Copying would copy all details about request so I can quickly share my request with anyone who is interested.

### Describe the solution you'd like

It would be nice if I could just right click on a request and copy or export it to a text file.  Copying would copy all details about request so I can quickly share my request with anyone who is interested.

### Describe alternatives you've considered

Going into very hiden option to export but this exports entire collection and makes it very cryptic and not helpful as the collections are all copyied and placed in a folder with cryptic name..   Plus, I dont want to copy entire collection but only one single request.

Curl also is not helpful as the person may not be technical or cannot execute curl on their machine but has postman and is familiar in creating requests

**Comment by grinderX19** [comment]
Agreed, sometimes you just need to export one **or several** requests but not a full collection, would be good to have this feature.

**Comment by peppbaggio** [comment]
Fully agree. It would be very helpful. Any chance?

**Comment by Gleeble** [comment]
This would definitely be a great feature to have and doesn't really seem like it would be much work to implement it as you already have export behaviors available.

Some additional options for being able to export singly (or at least less than all):
* Allow for selecting which calls to export when exporting a collection
* Add a "Postman" option to the Code types so you could copy the necessary JSON for importing into another postman instance

**Comment by dhempy** [comment]
Exporting a full request (or series of requests?) as a HAR file would allow documentation of the request headers and body, and the response headers and body. This would be the ideal solution for a miriad of documentation/troubleshooting needs. 

I hope you will add a "Export as HAR" feature to Postman.
